docs/src/user-guide/guides-using-presto.md

```
@@ -143,13 +152,31 @@ Each dataset in CLP shows up as a table in Presto. To show all available dataset
 
 ```sql
-SELECT * FROM default LIMIT 1;
+SELECT user FROM default LIMIT 1;
 ```
 
+:::{warning}
+`SELECT *` currently causes a crash due to a [known issue][y-scope/velox#27]. This will be resolved
+soon. See the [limitations](#limitations) section for all current limitations.
+:::
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)